docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-authoring-ha-modify-proc.adoc

```diff
@@ -0,0 +1,45 @@
+[id='environment-authoring-ha-modify-proc']
+= Modifying the template for the High Availability authoring environment
+
+By default, the high-availability authoring template creates a MySQL pod to provide the database server for the {KIE_SERVER}. If you prefer to use PostgreSQL or to use an external server (outside the OpenShift project), you need to modify the template before deploying the environment.
+
+You can also modify the High Availability authoring template to change the number of replicas initially created for {CENTRAL}.
+
+.Procedure
+
+Edit the `rhpam-70-authoring-ha.yaml` template file to make any of the following changes as necessary.
+
+* If you want to use PostgreSQL instead of MySQL, replace several sections of the file, marked with comments, with sections from the `rhpam70-kieserver-postgresql.yaml` file that are also marked with comments. All serctions are marked from `BEGIN` to `END`:
+
+. Replace the section named `MySQL database parameters` with the section named `PosgrreSQL database parameters`. (Take this section and all subsequent replacement sections from the `rhpam70-kieserver-postgresql.yaml` file.)
+. Replace the section named `MySQL service` with the section named `PosgrreSQL service`.
+. Replace the section named `MySQL driver settings` with the section named `PosgreSQL driver settings`.
+. Replace the section named `MySQL deployment config` with the section named `PosgreSQL deployment config`.
+. Replace the section named `MySQL persistent volume claim` with the section named `PosgreSQL persistent volume claim`.
+* If you want to use an external database server:
```

Comment:
"server, make the following changes:" Just per IBM SG, about not using incomplete sentences with a colon. I personally don't care, just letting you know our style perscription.